### ISSUE TITLE
Objects inside CanvasLayers are not visible in camera preview

### ISSUE THREAD

**Issue by Araraura** [anthonyec/godot_little_camera_preview#1]
I downloaded this addon and put it in my game, however it seems that the camera preview doesn't show any sprites or other objects that are children of a CanvasLayer (see image)

![image](https://github.com/anthonyec/godot_little_camera_preview/assets/25017422/65438796-4262-4d63-a799-c6f15e0835d3)

**Comment by anthonyec** [comment]
Good catch! I haven't worked much with canvas layers, but I'll look into it and see if there's a way to include them
